lib/node_modules/@stdlib/blas/ext/base/sapx/src/addon.c

```diff
@@ -0,0 +1,56 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/blas/ext/base/sapx.h"
+#include <node_api.h>
+#include "stdlib/napi/export.h"
+#include "stdlib/napi/argv_double.h"
+#include "stdlib/napi/argv.h"
+#include "stdlib/napi/argv_int64.h"
+#include "stdlib/napi/argv_strided_float32array.h"
+
+
+/**
+* Adds a constant to each element in a single-precision floating-point strided array.
+*
+* ## Notes
+*
+* -   When called from JavaScript, the function expects four arguments:
+*
+*     -   `N`: number of indexed elements
+*     -   `alpha`: scalar
+*     -   `X`: input array
+*     -   `strideX`: `X` stride length
+*/
+static napi_value addon( napi_env env, napi_callback_info info ) {
+	STDLIB_NAPI_ARGV( env, info, argv, argc, 4 );
+	//napi_status status;
+
+	STDLIB_NAPI_ARGV_INT64(env, N, argv, 0);
+	STDLIB_NAPI_ARGV_DOUBLE(env, alpha, argv, 1);
+	STDLIB_NAPI_ARGV_INT64(env, strideX, argv, 3);
+ 
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT32ARRAY(env, X, N, strideX, argv, 2)
+
+	c_sapx( N, (float)alpha, (float *)X, strideX );
+
+	return NULL;
+}
+
+STDLIB_NAPI_MODULE_EXPORT_FCN(addon)
+//NAPI_MODULE( NODE_GYP_MODULE_NAME, Init )
```

Comment:
```suggestion
STDLIB_NAPI_MODULE_EXPORT_FCN( addon )
```

Don't leave in stray commented out code.